skyvern/webeye/browser_factory.py

```diff
@@ -89,7 +89,13 @@ async def create_browser_context(
             creator = cls._creators.get(browser_type)
             if not creator:
                 raise UnknownBrowserType(browser_type)
-            return await creator(playwright, **kwargs)
+            if browser_type == "chromium-attached":
```

Comment:
I don't think you need this if statement -- I think you can move the `ws_endpoint` code into `_create_Attached_chromium`

Reply:
In hindsight, that seems obvious 👍